media_driver/agnostic/common/os/mos_sw_swizzle.c

```diff
@@ -0,0 +1,1029 @@
+/*
+* Copyright (c) 2019, Intel Corporation
+*
+* Permission is hereby granted, free of charge, to any person obtaining a
+* copy of this software and associated documentation files (the "Software"),
+* to deal in the Software without restriction, including without limitation
+* the rights to use, copy, modify, merge, publish, distribute, sublicense,
+* and/or sell copies of the Software, and to permit persons to whom the
+* Software is furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included
+* in all copies or substantial portions of the Software.
```

Comment:
License looks similar between media-driver and mesa, but there are differences. For example, media driver miss "(including the next paragraph)" words. Please, check what you should do in this case.